README.md

```diff
@@ -33,620 +33,677 @@ If you want to build the CLI using the source code, please, take into account th
+5aabb75a1de0240abb000187   Windows 2012 R2   {"memory":4096}        5aabb7551de0240abb000066
+5aabb75a1de0240abb000188   Joomla            {"memory":1024}        5aabb7551de0240abb000064
+5aabb75a1de0240abb000189   Magento           {"memory":1024}        5aabb7551de0240abb000064
+5aabb75b1de0240abb00018a   MongoDB           {}                     5aabb7551de0240abb000064
+5aabb75b1de0240abb00018b   Wordpress         {"memory":1024}        5aabb7551de0240abb000064
+5aabb75b1de0240abb00018c   Docker            {"memory":2048}        5aabb7551de0240abb000064
 ```
+
+You can choose whatever application/stack is fine for your purpose, we choose `Wordpress`. Take note of the application identifier, `5aabb75b1de0240abb00018b` for `Wordpress`.
+
+We will also need the location where we want our server to be instantiated. Execute `concerto wizard locations list` to get the possible locations and its identifier.
+
+```bash
 $ concerto wizard locations list
-ID                          NAME
-53f0f0e2d8a5975a1c00006b    USA
-53f0f0e2d8a5975a1c00006c    Europe
-53f0f0e2d8a5975a1c00006d    Asia Pacific
-53f0f0e2d8a5975a1c00006e    South America
+ID                         NAME
+5aabb7551de0240abb000060   North America
+5aabb7551de0240abb000061   Europe
+5aabb7551de0240abb000062   Asia Pacific
+5aabb7551de0240abb000063   South America
+```
+
+Take note of your preferred location. We will use `5aabb7551de0240abb000060` for `North America`.
+
+When using IMCO's Web UI, the wizard takes care of filtering appropriate cloud accounts for that provider and location. However, using the CLI is the user's responsibility to chose a provider cloud account for that application/stack and location; and a server plan capable of instantiating the stack in that location.
+To show all possible cloud accounts execute this command:
+
+```bash
+$ concerto wizard cloud_providers list --app_id 5aabb75b1de0240abb00018b --location_id 5aabb7551de0240abb000060
+ID                         NAME                  REQUIRED_CREDENTIALS
+5aabb7511de0240abb000001   AWS                   [access_key_id secret_access_key]
+5aabb7511de0240abb000002   Mock                  [nothing]
+5aabb7511de0240abb000004   Microsoft Azure ARM   [tenant_id client_id secret subscription_id]
+5aabb7511de0240abb000005   Microsoft Azure       [tenant_id subscription_id]
 ```
-Take note of your preferred location. We will use `53f0f0e2d8a5975a1c00006c` for Europe.
 
-When using Concerto's Web UI, the wizard takes care of filtering appropriate cloud providers for that location. However, using the CLI is the user's responsibility to chose a cloud provider and a server plan capable of instantiating the stack in that location. To show all possible providers execute this command:
+Take also into account that you should have configured your credentials before, using the Web UI or `concerto settings cloud_accounts create`. We will choose `Microsoft Azure`, whose ID is `5aabb7511de0240abb000005`.
```

Comment:
See comment above on cloud account management. Specifically, `concerto settings cloud_accounts create` should not longer be available...